server/src/main/java/org/elasticsearch/index/query/MatchPhraseQueryBuilder.java

```diff
@@ -39,6 +40,7 @@
 public class MatchPhraseQueryBuilder extends AbstractQueryBuilder<MatchPhraseQueryBuilder> {
```

Comment:
The additions in this class are a bit messy, but I favored consistency with `MatchQueryBuilder` and `MultiMatchQueryBuilder`.